### PR TITLE
HTML: Fix incorrect test for U+000C in window.open() features

### PR DESCRIPTION
**Pull request by zcorpan**
Found by cdumez in https://github.com/w3c/web-platform-tests/pull/5306#discussion_r113521462

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5715)

### CODE REVIEW

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5715/)

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226312328)

# Lint #

Passed

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226312330)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 725d1f04cdc11887c93da43dfc33e281a895cff0
Using browser at version BuildID 20170426100344; SourceStamp 0f5ba06c4c5959030a05cb852656d854065e2226
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5715/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html">/html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html</a></summary>


|                        Subtest                         | Results |                   Messages                  |
|--------------------------------------------------------|---------|---------------------------------------------|
|                                                        | OK      |                                             |
| `HTML: window.open features: tokenization -- noopener` | FAIL    | `SyntaxError: missing ] after element list` |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/226312338)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 725d1f04cdc11887c93da43dfc33e281a895cff0

**Comment by cdumez** [comment]
Hmm, I get:
SyntaxError: Unexpected string literal 'foo=\u000Cnoopener'. Expected either a closing ']' or a ',' following an array element.

File: html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html [hunk 1]
```diff
@@ -140,6 +140,7 @@
     'no,opener', //     => ('no', ''), ('opener', '')
     '\0noopener', //    => ('\0noopener', '')
     'noopener\u0000=yes' // => ('noopener\0', 'yes')
```
Comment (cdumez) [inline html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html]: Missing a comma at the end here.

**Comment by cdumez** [comment]
Fixing via https://github.com/w3c/web-platform-tests/pull/5721

File: html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html (hunk 1 above)
Comment (zcorpan) [inline html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html]: Sigh, sorry!

**Pull request merged**